python/cudf/cudf/core/dataframe.py

```diff
@@ -1374,6 +1374,10 @@ def fallback(col, fn):
         elif isinstance(other, (numbers.Number, cudf.Scalar)):
             for col in self._data:
                 result[col] = op(self[col], other)
+        elif isinstance(other, np.ndarray) and other.ndim == 0:
+            other = other.item()
+            for col in self._data:
+                result[col] = op(self[col], other)
```

Comment:
Can we add the `np.ndarray` instance to the above group now that the normalization would be handled at the Column level anyway?